saleor/graphql/views.py

```diff
@@ -29,18 +31,11 @@
 
 
 def tracing_wrapper(execute, sql, params, many, context):
-    with opentracing.global_tracer().start_span(operation_name="query") as span:
-        span.set_tag("component", "db")
-        span.set_tag("db.statement", sql)
-        try:
-            result = execute(sql, params, many, context)
-        except Exception as e:
-            span.set_tag("error", True)
-            span.set_tag("error.object", e)
-            raise
-        else:
-            span.set_tag("error", False)
-            return result
+    with ot.global_tracer().start_active_span(operation_name="query") as scope:
```

Comment:
By the way, this change removes `error` but it is normal. OpenTracing is already handling this behind the scene.